### PR TITLE
feat(i18n-RU): Update Russian translations

### PR DESCRIPTION
**Pull request by V-Zemlyakov**
Updating russian translation.
Signed-off-by: V.Zemlyakov zemlyakov@bmstu.ru

### CODE REVIEW

**Comment by pedrolamas** [comment]
Thank you @V-Zemlyakov, can you please add a footer to the main PR comment with a "Signed-off-by" line, following the [CONTRIBUTING](https://github.com/fluidd-core/fluidd/blob/develop/CONTRIBUTING.md) guidelines.

Thank you!

**Review by pedrolamas — APPROVED** [review]

**Review by matmen — APPROVED** [review]

File: src/locales/ru.yaml [hunk 1]
```diff
@@ -444,71 +545,101 @@ app:
...
       check_for_updates: Проверить обновления
       finish: Закончено
       update: Обновление
+      update_all: Обновить всё
       view_versions: Новые версии
     label:
       commit_history: История коммитов
       commits_on: Коммиты на
-      committed: Закоммичено
+      committed: Закомментировано
```
Comment (Chuvi-w) [inline src/locales/ru.yaml:642]: Глагол "to commit" означает "совершать/ делать вклад". Он не имеет общего с "to comment" - "комментировать".

//Wrong translation. "Commited" translated as "Commented".

**Review by Chuvi-w — COMMENTED** [review]
Translation issue

File: src/locales/ru.yaml (hunk 1 above)
Comment (pedrolamas) [reply src/locales/ru.yaml:642]: So was the original "committed: Закоммичено" correct?

Comment (V-Zemlyakov) [reply src/locales/ru.yaml:642]: Everything related to "git" is better not to be translated at all. Leave original. Whoever needs to understand, and whoever does not understand, does not need to. IMHO
(autotranslate)

Comment (Chuvi-w) [reply src/locales/ru.yaml:642]: > So was the original "committed: Закоммичено" correct?
Yes, partially. It's a borrowed world, a calque from english, which is not exist in official russan. Thouse who use git understands it's meaning, who don't use git or other cvs may not understand.

Comment (Chuvi-w) [reply src/locales/ru.yaml:642]: > So was the original "committed: Закоммичено" correct?
Yes, partially. It's a borrowed world, a calque from english, which is not exist in official russan. Thouse who use git understands it's meaning, who don't use git or other cvs may not understand.

Comment (pedrolamas) [reply src/locales/ru.yaml:642]: I have no personal opinion on this (I leave translations to native speakers and accept Translation PRs as fact).

If you want to send a PR to correct this (just click [here](https://github.com/V-Zemlyakov/fluidd/blob/develop/src/locales/ru.yaml) then click the "edit" to change the file and send a PR) I will integrate the changes.

**Pull request merged**